Doppler.HtmlEditorApi/DopplerFieldsProcessor.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Doppler.HtmlEditorApi;
+
+/// <summary>
+/// This class deals with the conversion between Doppler field-names
+/// to Doppler field-ids and vice versa.
+/// </summary>
+public class DopplerFieldsProcessor
+{
+    private readonly ReadOnlyDictionary<string, int> FieldIdsByNameOrAlias;
+
+    private readonly ReadOnlyDictionary<int, string> FieldNamesById;
```

Comment:
Just to know, why don't these properties have the `_` prefix?
```suggestion
    private readonly ReadOnlyDictionary<string, int> _fieldIdsByNameOrAlias;

    private readonly ReadOnlyDictionary<int, string> _fieldNamesById;
```

Reply:
Good catch, but it is out of the scope of this PR, you should found it before :P

Reply:
I was wrong, it is the right PR, fixing...

Reply:
Done, resolving conversation...